python/lsst/obs/monocam/monocam.py

```diff
@@ -162,68 +181,64 @@ def _makeAmpInfoCatalog(self):
             for ampX in range(8):
-                record = ampCatalog.addNew()
-                record.setName("%d%d" % (ampX, ampY))
+                amplifier = cameraGeom.Amplifier.Builder()
+                amplifier.setName("%d%d" % (ampX, ampY))
 
                 if bool(ampY):
-                    record.setBBox(geom.Box2I(
+                    amplifier.setBBox(geom.Box2I(
                         geom.Point2I(ampX*xDataExtent, ampY*yDataExtent),
                         geom.Extent2I(xDataExtent, yDataExtent),
                     ))
                 else:
-                    record.setBBox(geom.Box2I(
+                    amplifier.setBBox(geom.Box2I(
                         geom.Point2I((7 - ampX)*xDataExtent, ampY*yDataExtent),
                         geom.Extent2I(xDataExtent, yDataExtent),
                     ))
 
-                readCorner = LL  # in raw frames; always LL because raws are in amp coords
+                # in raw frames; always LL because raws are in amp coords
+                readCorner = cameraGeom.ReadoutCorner.LL
                 # bias region
                 x0Bias = extended + xDataExtent
                 y0Data = 0
                 x0Data = extended
 
-                record.setRawBBox(geom.Box2I(
+                amplifier.setRawBBox(geom.Box2I(
                     geom.Point2I(0, 0),
                     geom.Extent2I(xRawExtent, yRawExtent),
                 ))
-                record.setRawDataBBox(geom.Box2I(
+                amplifier.setRawDataBBox(geom.Box2I(
                     geom.Point2I(x0Data, y0Data),
                     geom.Extent2I(xDataExtent, yDataExtent),
                 ))
-                record.setRawHorizontalOverscanBBox(geom.Box2I(
+                amplifier.setRawHorizontalOverscanBBox(geom.Box2I(
                     geom.Point2I(x0Bias, y0Data),
                     geom.Extent2I(h_overscan, yDataExtent),
                 ))
-                record.setRawVerticalOverscanBBox(geom.Box2I(
+                amplifier.setRawVerticalOverscanBBox(geom.Box2I(
                     geom.Point2I(x0Data, y0Data+yDataExtent),
                     geom.Extent2I(xDataExtent, v_overscan),
                 ))
-                record.setRawXYOffset(geom.Extent2I(ampX*xRawExtent, ampY*yRawExtent))
-                record.setReadoutCorner(readCorner)
-                record.setGain(self.gain[(ampX, ampY)])
-                record.setReadNoise(self.readNoise[(ampX, ampY)])
-                record.setSaturation(saturation)
-                record.setHasRawInfo(True)
-                record.setRawFlipX(bool(ampY))
+                amplifier.setRawXYOffset(geom.Extent2I(ampX*xRawExtent, ampY*yRawExtent))
+                amplifier.setReadoutCorner(readCorner)
+                amplifier.setGain(cls.gain[(ampX, ampY)])
+                amplifier.setReadNoise(cls.readNoise[(ampX, ampY)])
+                amplifier.setSaturation(saturation)
+                # amplifier.setHasRawInfo(True)
```

Comment:
Did you mean to leave this comment in the code?

Reply:
Removed.  This isn't used and needs to be updated in a future ticket (DM-21277).